maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/pom.xml

```diff
@@ -44,6 +44,17 @@
       <groupId>org.apache.maven.scm</groupId>
       <artifactId>maven-scm-provider-git-commons</artifactId>
     </dependency>
+    <dependency>
+      <artifactId>plexus-interactivity-api</artifactId>
+      <groupId>org.codehaus.plexus</groupId>
```

Comment:
swap groupId and artifactId

Reply:
Done in https://github.com/apache/maven-scm/pull/154/commits/617c43e02f33474dc9fc2c17ee400f874d18d789